File: backend/src/main/java/com/cm/ecohub/constants/ApiPaths.java
package com.cm.ecohub.constants;

import lombok.NoArgsConstructor;

@NoArgsConstructor
public final class ApiPaths {
    public static final String API_BASE = "/api";

    public static final String AUTH = API_BASE + "/auth";
    public static final String AUTH_LOGIN = AUTH + "/login";
    public static final String AUTH_REFRESH = AUTH + "/refresh";
    public static final String AUTH_LOGOUT = AUTH + "/logout";
}

File: frontend/src/constants/apiPaths.js
export const API_PATHS = {
    AUTH_LOGIN: '/auth/login',
    AUTH_REFRESH: '/auth/refresh',
    AUTH_LOGOUT: '/auth/logout',

    USERS: '/users',
    USER_BY_ID: (id) => `/users/${id}`,
    ME: '/me',

    AI: '/ai',

    VILLAGES: '/villages',
    VILLAGE_BY_ID: (id) => `/villages/${id}`,
};